### DISCUSSION TITLE
when we use Linkerd2 do multi k8s cluster, Is LoadBalance a must?

### DISCUSSION THREAD

**Discussion by liuxuzxx** [linkerd/linkerd2#10337] [Help] [answered]
Now I want to use Linkerd2 to communicate across k8s clusters. I see a service that supports the LoadBalance type. Is this a necessary condition? If the current cluster only supports the form of NodePort, is it also possible?

**Comment by alpeb** [comment]
Yep, you can use services of type NodePort as well, via the `gateway.serviceType=NodePort` setting. You can also set the port via `gateway.nodePort`. These can be set as `values.yaml` entries in the [helm chart](https://artifacthub.io/packages/helm/linkerd2/linkerd-multicluster).

**Reply by liuxuzxx** [reply]
thanks，very thanks！

**Comment by liuxuzxx** [comment, marked as answer]
Can the Linkerd2 cross-cluster solution achieve Pod cross-cluster access through ip?  For example,We have two k8s cluster ,one is west and other is east!   when we in west pod west-pod'ip is: 10.0.2.3   the east cluster pod ip is:192.168.0.2    can we in west pod:use curl http://192.168.0.2:5000/helloworld?

**Reply by alpeb** [reply]
Linkerd's multicluster approach doesn't require a flat network, i.e. we don't require all the clusters to share the same network address space. Please refer to [this article](https://linkerd.io/2020/02/17/architecting-for-multicluster-kubernetes/index.html) explaining the reasoning behind the architecture we designed for achieving multicluster connectivity, and these two pages in the docs giving more details about how to set things up:
https://linkerd.io/2.12/features/multicluster/
https://linkerd.io/2020/02/17/architecting-for-multicluster-kubernetes/index.html
In short, to initiate a connection to a pod living in another cluster you should target a service mirror service located in the same cluster as your client. That service gets created automatically after you have linked both clusters as explained in those docs.
